services/motion/builtin/builtin.go

```diff
@@ -115,13 +120,19 @@ func (ms *builtIn) Reconfigure(
 			movementSensors[name] = dep
 		case slam.Service:
 			slamServices[name] = dep
+		case vision.Service:
+			visionServices[name] = dep
+		case camera.Camera:
```

Comment:
this likely go away, once we decide how to specify which cameras a service uses though the nav cfg